### ISSUE TITLE
Application crashes immediatly after execution

### ISSUE THREAD

**Issue by Fouine91** [Dom2364/PlaneAlerter#5]
An issue solved for information...

Issue :
After configuring options, as soon as PlaneAlerter tries to connect to VRS it crash with a .NET error.
And then each time it is started.

Solution:
In my concern, my VRS installation is on a Windows 7 Pro French VM, and after many tries, no way to get it work... Then, i installed a Win7 Pro UK VM, just installed Plane Alerter (and necessary system software), and now it works like a charm, UP for almost 5 months non-stop.

In conclusion, i can tell that Plane Alerter only works on UK systems ;) (at least Win7).

Thanks for this useful tool.
Cheers.
Thibaut

**Comment by Dom2364** [comment]
Not sure why the OS language would cause the program to crash, I would need the .NET error info to be able to debug it properly

**Comment by Fouine91** [comment]
I made a test yesterday: changed regional settings to french on the english VM (date, time, and decimals), got the crash at program startup... Everything ok when changing back to English (US) preset.
I will reinstall the program on my french VM, and changing regional settings to see if this solve the problem.
It doesn't matter for me to be on a English OS, but this info could be interesting for people ;)
I'll tell the results soon.
Regards

**Comment by Fouine91** [comment]
Hi,
I finally tested on the machine where planealerter crashes, and starting from the regional settings idea...
So, on the french Windows 7 Pro, default regional settings (french of course) make the program crash as soon it connects to VRS, but if i change the regional settings to English (US) the program no longer crashes... So i can use it on the same machine as VRS.
Hope it could help other if same case occurs ;)
Cheers.

**Comment by Dom2364** [comment]
Just fixed it, seems like it was crashing when decoding aircraft position trails from VRS and the OS was using a region setting that used commas as decimal separators. I'll be releasing an update in the next few days with the fix, thanks for reporting this bug!

**Comment by Fouine91** [comment]
You're welcome, and thanks for you hard work :)
